swagger/reflect.go

```diff
@@ -73,12 +76,36 @@ func inspect(t reflect.Type, jsonTag string, formatTag string) Property {
 			}
 		}
 
+		var err error
+		for _, val := range []string{minLenTag, maxLenTag} {
+			if val != "" {
+				if val == minLenTag {
+					p.MinLength, err = strconv.Atoi(minLenTag)
+				} else {
+					p.MaxLength, err = strconv.Atoi(maxLenTag)
```

Comment:
This won't work; if you put `min_length="5" max_length="5"` it'll actually be interpreted as `min=5 min=5`, i.e. no max. Handle them in two separate if statements.